dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts

```diff
@@ -113,7 +118,11 @@ export default class DigitalTwinDetailComponent extends mixins(EvanComponent) {
         }
 
         // initialize twin structure
-        await newTwin.initialize();
+        await newTwin.initialize()
```

Comment:
Please use try catch.

Reply:
Why do we need a try catch here? The promise catch handler seems more best practice and easier to reason about.

Reply:
Just wanted to reduce the style of `then` and `catch`, but is okay at this point.